mysite/settings.py

```diff
@@ -38,7 +38,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [os.path.join(BASE_DIR, 'templates')],
```

Comment:
Why do you put templates outside of the django app?

Reply:
I have no clue. It's as I did it before. If it isn't ok I can move to the app :D

Reply:
There is also some file that belongs to whole project as `base.html` and `components` dir so I would like to left all templates as it is for simplicity.

Reply:
But you don't have any other apps anyway. It looks like you're overcomplicating things.